hack/verify-test-code.sh

```diff
@@ -25,7 +25,7 @@ mapfile -t all_e2e_files < <(find test/e2e -name '*.go' | grep -v 'test/e2e/fram
 errors_expect_no_error=()
 for file in "${all_e2e_files[@]}"
 do
-    if grep "Expect(.*)\.NotTo(.*HaveOccurred()" "${file}" > /dev/null
+    if grep -E "Expect\(.*\)\.(NotTo|ToNot)\(.*HaveOccurred\(\)" "${file}" > /dev/null
```

Comment:
verify-test-code.sh adds to detect `Expect(err).ToNot(HaveOccurred())`.
But, it says about only `NotTo` as the below.
```
The above files need to use framework.ExpectNoError(err) instead of
Expect(err).NotTo(HaveOccurred()) or gomega.Expect(err).NotTo(gomega.HaveOccurred())
```
Why don't you add `ToNot` to error comment? Is it redundant?

Reply:
That is a nice point.
As we see, `ToNot` is not major and I feel it could be enough to contain `NotTo` only for easy explanation.

Reply:
OK, I understood that.
/lgtm